draft-ietf-quic-qcram.md

```diff
@@ -175,17 +197,21 @@ always non-zero; a zero index is reserved for literal names.
 ~~~~~~~~~~
 {: title="Insert Header Field -- Indexed Name"}
 
-Otherwise, the header field name is represented as a string literal (see Section
-5.2 of [RFC7541]). A value 0 is used in place of the table reference, followed
-by the header field name.
+
+### Insert Without Reference
+
+An addition to the header table where both the header field name and the header
+field value are represented as string literals (see {{primitives}}) starts with
+the '00' two-bit pattern.
+
+The name is represented as a 6-bit prefix string literal, while the value is
+represented as an 8-bit prefix string literal.
 
 ~~~~~~~~~~ drawing
      0   1   2   3   4   5   6   7
    +---+---+---+---+---+---+---+---+
-   | 1 | 0 |           0           |
-   +---+---+-----------------------+
-   | H |     Name Length (7+)      |
-   +---+---------------------------+
+   | 0 | 0 | H | Name Length (5+)  |
```

Comment:
Almost a bike shed: invert the second bit here.  Then you can reasonably say that you are describing 11 (insert with static name reference), 10 (insert with dynamic name reference), 01 (insert with new name), 00 (duplication).

Reply:
I'm not wholly sure that makes a difference, but I suppose there's an argument to be made that '00' is the "unique" one.